compiler/damlc/tests/src/DA/Test/ScriptService.hs

```diff
@@ -980,9 +980,23 @@ main =
                   , "  revoked === [CanActAs p1, CanReadAs p2]"
                   , "  rights <- listUserRights \"u1\""
                   , "  rights === []"
+                  , "testUserAlreadyExists = do"
+                  , "  u1 <- createUser (User \"u1\" None) []"
+                  , "  u2 <- createUser (User \"u1\" None) []"
+                  , "  pure ()"
+                  , "testUserNotFound = do"
+                  , "  u1 <- createUser (User \"u1\" None) []"
+                  , "  deleteUser \"u2\""
+                  , "  pure ()"
                   ]
                 expectScriptSuccess rs (vr "testUserManagement") $ \r ->
                     matchRegex r "Active contracts: \n"
+                expectScriptSuccess rs (vr "testUserRightManagement") $ \r ->
```

Comment:
This test is not directly related to this change; `testUserRightManagement` already existed, but the call was missing.

Reply:
duh, good catch